src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int randomIndex = new Random().nextInt(Colors.values().length);
```

Comment:
you should create a field for this `Random` instance, so that you don't create a new one each time this method is called:
```
private final Random random = new Random():
```